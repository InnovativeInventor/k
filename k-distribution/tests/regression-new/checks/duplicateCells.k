module DUPLICATERULES-SYNTAX
  imports DEFAULT-CONFIGURATION

    configuration <test> $TST:Int</test>

endmodule

module DUPLICATECELLS
    imports DUPLICATERULES-SYNTAX

    configuration <k> $PGM:Int </k>
                  <test> $TST:Int</test>

endmodule
