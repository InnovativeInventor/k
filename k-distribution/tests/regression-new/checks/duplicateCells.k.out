[Error] Compiler: Duplicate initialization of K cell found. Only one initialization is allowed.
	Source(duplicateCells.k)
	Location(11,23,11,31)
	11 |	    configuration <k> $PGM:Int </k>
	   .	                      ^~~~~~~~
[Error] Compiler: Duplicate initialization of Test cell found. Only one initialization is allowed.
	Source(duplicateCells.k)
	Location(12,26,12,34)
	12 |	                  <test> $TST:Int</test>
	   .	                         ^~~~~~~~
[Error] Compiler: Had 2 structural errors.
